### ISSUE TITLE
[feature] Specify parameter values for one Trial

### ISSUE THREAD

**Issue by gaocegege** [kubeflow/katib#892]
/kind feature

**Describe the solution you'd like**
[A clear and concise description of what you want to happen.]

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Comment by hougangliu** [comment]
can you detail more about what you plan to do for this? @gaocegege

**Comment by gaocegege** [comment]
I have no idea about it. But it is a useful feature, I think.

**Comment by gaocegege** [comment]
Ref https://optuna.readthedocs.io/en/latest/reference/trial.html#optuna.trial.FixedTrial

**Comment by hougangliu** [comment]
> Ref https://optuna.readthedocs.io/en/latest/reference/trial.html#optuna.trial.FixedTrial

Get it. Maybe it is useful when a user just want to compare a set of known hyperparameter combination

**Comment by gaocegege** [comment]
Yeah, but not sure how to support it in katib and do not know if it can help users a lot.
